### DISCUSSION TITLE
is there actually MSSQL support?

### DISCUSSION THREAD

**Discussion by louis-eyond** [stalwartlabs/smtp-server#48] [Q&A] [answered]
The smtp-server readme states MSSQL database support.  I can't find any other reference in source or documentation.  Is this a typo, or is there something I am missing?  Thank you.

**Comment by mdecimus** [comment, marked as answer]
There was support before but the SQL library we use stopped supporting it so we need to remove it from this repository's README (it is not listed in the official docs).
